examples/python3/BUILD

```diff
@@ -0,0 +1,16 @@
+load("@io_bazel_rules_docker//python:image.bzl", "py_image")
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
+
+py_image(
```

Comment:
Is it your expectation that we should change this to `py3_image` in the limit?

Reply:
In the absence of something more clever, yes.

(It's unfortunate that there is a circular dependency between these repositories.)